CHANGELOG.md

```diff
@@ -22,7 +22,8 @@ Version template:
 # Dynamic Extensions For Alfresco Changelog
 
 ## [2.0.6] - UNRELEASED
-
+### Fixed
+* [314](https://github.com/xenit-eu/dynamic-extensions-for-alfresco/issues/314) Improve documentation on `ScheduledTask.cluster`
```

Comment:
Thanks for your effort, however it's not necessary to add (small) documentation changes to the CHANGELOG. On the contrary: I would explicitly keep it out.

Reply:
Ill remove it again.